policy/score.go

```diff
@@ -16,6 +16,7 @@ const (
 	ScoreType_Unscored
 	ScoreType_OutOfScope
 	ScoreType_Disabled
+	ScoreType_Snoozed
```

Comment:
do you need to modify the function below?

Reply:
whoops... I probably should